src/main/frontend/util.cljc

```diff
@@ -1040,13 +1040,24 @@
 
 #?(:clj
    (defmacro with-time
-     "Evaluates expr and prints the time it took. Returns the value of expr and the spent time."
+     "Evaluates expr and prints the time it took. 
+      Returns the value of expr and the spent time of string in msecs."
      [expr]
      `(let [start# (cljs.core/system-time)
             ret# ~expr]
         {:result ret#
          :time (.toFixed (- (cljs.core/system-time) start#) 6)})))
 
+#?(:clj
+   (defmacro with-time-number
```

Comment:
Optional: If this is for tests only, frontend.test.helper is a good place for this

Reply:
I merged the `with-time-number` with `with-time`, as they only have different return type